migrations/versions/ff3141aecdd4_.py

```diff
@@ -0,0 +1,80 @@
+"""Update Olink's assay_raw_ct artifact data format from XLSX to CSV.
+
+Revision ID: ff3141aecdd4
+Revises: b8eaf567ac2f
+Create Date: 2019-11-14 11:43:49.297901
+
+"""
+import os
+from typing import Callable
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm.session import Session
+from google.cloud import storage
+
+from cidc_schemas.migrations import v0_10_0_to_v0_10_2, MigrationResult
+from cidc_api.models import TrialMetadata, DownloadableFiles
+from cidc_api.config.settings import GOOGLE_DATA_BUCKET
+
+# revision identifiers, used by Alembic.
+revision = "ff3141aecdd4"
+down_revision = "b8eaf567ac2f"
+branch_labels = None
+depends_on = None
+
+is_testing = os.environ.get("TESTING")
+
+session = Session(bind=op.get_bind())
+storage_client = storage.Client() if not is_testing else None
+
+
+def _do_metadata_migration(metadata_migration: Callable[[dict], MigrationResult]):
```

Comment:
I wonder if this code is generic enough that it should live in a separate module somewhere. It's possible we'll be able to use it directly for future metadata migrations.

Reply:
Yeah, would be nice.